README.md

```diff
@@ -7,31 +7,95 @@
 [![CI](https://github.com/code-pushup/cli/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/code-pushup/cli/actions/workflows/ci.yml?query=branch%3Amain)
 [![Codecov](https://codecov.io/gh/code-pushup/cli/branch/main/graph/badge.svg?token=Y7V489JZ4A)](https://codecov.io/gh/code-pushup/cli)
 
-🔎🔬 **Quality metrics for your software project.** 📉🔍
+<h1 align="center">🔎🔬Code quality integrations for any tool📉🔍</h1>
 
-1. ⚙️ **Configure what you want to track using your favourite tools.**
-2. 🤖 **Integrate it in your CI.**
-3. 🌈 **Visualize reports in a beautiful dashboard.**
+<h3>Code quality tools are like phone chargers. Everyone has a different plug.</h3>
+
+- **Incomplete or missing tracking** due to a mix of portals and custom tools is costly and unreliable
+- **No Standards** in data and processes leads to flaky, self-maintained zombies
+- **Results are spread** over multiple places and need manual steps
+
+<h2>Standards for code quality that integrates any tool</h2>
+
+<h3>Automated CI Integration</h3>
+
+- **Seamless integration** into any CI pipeline with full control.
+- Our tool **works on all platforms** like GitHub, Gitlab, Bitbucket etc.
+- Get updates [on every PR](./packages/cli/README.md#-ci-automation)!
```

Comment:
I would link to the GitHub Action instead (includes PR feedback): https://github.com/marketplace/actions/code-pushup

Reply:
linked